glossary.md

```diff
@@ -295,6 +297,7 @@ Nếu bạn cho rằng một từ không nên dịch ra tiếng Việt, bạn c
 |-----------------------------------|-------------------------------------------|----------------------------------------------|
 | random variable                   | biến ngẫu nhiên                           |                                              |
 | recall                            | recall                                    |                                              |
+| receptive field (CNN) | vùng thụ cảm | |
```

Comment:
```suggestion
| receptive field (CNN) | trường tiếp nhận | |
```
Thế này thì sao?

Reply:
Tin học Kỹ thuật Khoa học Cơ khí điện tử máy móc IT guy hơn rồi ....

Reply:
mình tưởng từ này có gốc y khoa nên dịch thế :smile:
anw mình nghĩ vẫn nên dịch field là `vùng`, chứ không phải `trường` giống trong database hay form
```suggestion
| receptive field (CNN) | vùng tiếp nhận | |
```